scripts/joinmarket-qt.py

```diff
@@ -1460,17 +1460,22 @@ def restartWithMsg(self, msg):
         self.close()
 
     def recoverWallet(self):
-        success = wallet_generate_recover_bip39("recover", "wallets",
-                                                "wallet.jmdat",
-                                                callbacks=(None, self.seedEntry,
+        try:
+            success = wallet_generate_recover_bip39("recover", "wallets",
+                                                    "wallet.jmdat",
+                                                    callbacks=(None, self.seedEntry,
                                                            self.getPassword,
                                                            self.getWalletFileName))
-        if not success:
-            JMQtMessageBox(self,
+            if not success:
+                JMQtMessageBox(self,
                            "Failed to recover wallet.",
                            mbtype='warn',
                            title="Error")
+                return
+        except Exception as e:
```

Comment:
Could you be more specific with the exception type being handled here? As the code is now it will catch a very wide range of errors and may hide bugs. For example, an exception caused when a programmer misspells a variable name e.g. `wallet.seeed` instead of `wallet.seed` will be caught here.

How about catch just StorageError ?

Reply:
I just did the same as in #273 here. If we change that, both should be changed.

Reply:
Fair enough, it should be fixed in another PR not this one.